test/seneca.test.js

```diff
@@ -378,14 +381,17 @@ describe(__filename, function(){
         assert.ok( o.foo.entry )
 
         si.add({op:'foo'},bar)
+				// console.log('=============')
         si.act('op:foo,a:1',function(e,o){
           assert.ok(gex('1~2~Seneca/0.6.*'+'/*').on(''+o.a+'~'+o.b+'~'+o.s))
           assert.ok( o.bar.entry )
-          assert.ok( !o.foo.entry )
+          // assert.ok( !o.foo.entry )
```

Comment:
intentionally removed?

Reply:
yes - optimized priors can't track if they are the "entry" into a series of seneca action calls - not that useful in practice and replaced with a new tracking system in the current Seneca